msteams-platform/tabs/how-to/authentication/tab-sso-register-aad.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Register your tab app with Azure AD
+description: Describes registering your tab app with Azure AD
+ms.topic: how-to
+ms.localizationpriority: medium
+keywords: teams authentication tabs Microsoft Azure Active Directory (Azure AD) access token SSO tenancy scope 
+---
+# Register your app in Azure AD
+
+Azure AD provides access to your tab app based on the app user's Teams identity. You'll need to register your tab app with Azure AD so that the app user who has signed into Teams can be given access to your tab app.
+
+## Enabling SSO on Azure AD
+
+Registering your tab app in Azure AD and enabling it for SSO requires making app configurations, such as generating app ID, defining API scope, and configuring the On-behalf-of (OBO) flow.
+
+:::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/register-azure-ad.png" alt-text="Configure Azure AD to send access token to Teams Client app" border="false":::
+
+Create a new app registration in Azure AD, and expose its (web) API using scopes (permissions). Configure a trust relationship between the exposed API on Azure AD and your app. The OBO flow allows app users to access your tab app without any further need of consent. You can add client IDs for the trusted mobile, desktop, and web applications that you want to pre-authorize.
+
+You may also need to configure additional details, such as authenticating app users on the platform or device where you want to target your app. Azure AD configuration enables SSO for your tab app in Teams. It responds with an access token for validating the app user.
+
+User-level Graph API permissions are supported, that is, email, profile, offline_access, and OpenId. If you require access to additional Graph scopes, such as `User.Read` or `Mail.Read`, see [Get an access token with Graph permissions](tab-sso-graph-api.md).
+
+> [!NOTE]
+> The Microsoft Teams Toolkit registers the Azure AD application in an SSO project.
+
+### Before you register with Azure AD
+
+It's helpful if you learn about the configuration for registering your app on Azure AD beforehand. Ensure that you've prepared to configure the following details prior to registering your app:
+
+- **Single- or multi-tenant options**: Your app can be line-of-business (LOB) app, public app, or software-as-a-service (SaaS) application. The tenancy options may differ based on the type of your app and how you want to distribute it.
+- **App platform**: Note the platform where your app is available. For tab apps, you can choose from web or Single-page application (SPA) platform. Mobile and desktop applications isn't available. It also includes noting the URL from where your app is accessible.
```

Comment:
This implies that the app won't work on a mobile client - but it will!
This Azure AD setting is used for certain OAuth 2.0 flows, but here the Microsoft Teams client hides that fact and uses the OBO Flow. So the app platform setting is not needed - at least I don't think so and I tested it without an app platform setting or redirect URI.
I'll bring this up in chat but tentatively suggest removing this bullet entirely.